src/centralstore/incremental/RelationBlock.h

```diff
@@ -0,0 +1,72 @@
+#include <cstring>
```

Comment:
We need to put license header here

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/115/commits/ed31ba0af33e9629173808960eef3cfe90c5dbe9

Reply:
Done